pennylane_forest/device.py

```diff
@@ -266,6 +266,7 @@ def apply_circuit_operations(self, operations):
 
     def reset(self):
         """Resets the device after the previous run."""
+        # pylint:disable=attribute-defined-outside-init
```

Comment:
As I mentioned earlier: if we use
```python
self.prog = Program()
self._state = None
```
instead of calling `reset()` in the `__init__` function, this pylint error won't happen.

Reply:
Okay, initialized those in `__init__`